README.md

```diff
@@ -74,13 +74,13 @@ Every WebAssembly chaincode should implement ```init``` function.
 ### WASMCC functions available to initiate transactions
 
 
-WASMCC have three functions available to initiate transaction from clients, namely, ```create```, ```invoke``` and ```installedChaincodes``` functions for a transaction
+WASMCC have three functions available to initiate transaction from clients, namely, ```create```, ```execute``` and ```installedChaincodes``` functions for a transaction
```

Comment:
you can do a single back tick instead of three.

Single is typically used for in line, and the triple back tick is meant for blocks of code.